docker/dev/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM lychee-base:latest AS base
+
+# Install development dependencies
+
+RUN \
+	apt-get install -y git composer nodejs
+
+RUN \
```

Comment:
Is that necessary ? I am not even sure that reduces the size of the image due to layering.

Reply:
there was no intention of reducing the image size, just so that only dev image contains composer, git, nodejs etc..